client/editSchedule.js

```diff
@@ -110,10 +129,10 @@ setupCompetitionCalendar = function(template, $calendarDiv, $editModal) {
         $editModal.modal('show');
       },
       eventDrop: function(calEvent, delta, revertFunc, jsEvent, ui, view) { // Existing entry dragged
-        eventChanged(calEvent);
+        eventChanged(calEvent, revertFunc);
```

Comment:
Yeah, I think we catch all cases, and if we don't, that should be handled in validation.

What I hate about these error messages is that there is no way to be specific. Instead of *The 2x2x2 final starts at 9:00*, we have to say *There are events earlier in the day*.